keyboards/AT101_Blackheart/AT101_Blackheart.h

```diff
@@ -0,0 +1,34 @@
+#ifndef AT101_Blackheart_H
+#define AT101_Blackheart_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
Two things here:
* Please change `KEYMAP` to `LAYOUT` here 
* this macro is "wrong".  You should reformat it so that it looks like the actual keyboard, rather than what the grid looks like. 

Specially, the macro here is so that you can convert the physical (electric) matrix into something "human friendly" (eg, the visual layout).

A good example of this is the clueboard 66's matrix: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/66/rev3/rev3.h#L33-L50

And to clarify, the positions don't have to match in location, just in where they end up in the matrix. 

So, for instance, the top like would: 
```
#define LAYOUT( \
  K000, K0100, K001, K0101, K0002, K0102, K0003, K0103, K0004, K0104, K0005, K0105, K0006, K0106, K0007, K0107, \
```

That would be KC_ESC, to KC_PAUS on the first row.  

Reformating this may be tedious, but would be a good idea. 

If you need help figuring this out, it may be worth dropping this into your `process_record_user` function, to tell which position is which:
```c
  xprintf("KL: row: %u, column: %u, pressed: %u\n", record->event.key.row, record->event.key.col, record->event.pressed);
```
And make sure that CONSOLE_ENABLE is set to "yes", and that you hook up HID Listen (you can use the QMK Toolbox on Windows or macOS to do this).

Reply:
Thank you for this! I had no idea that was possible and really helps clean up the layout. Will fix this right away.

Reply:
Also, would the section in brackets for the layout need to also be formatted in the same manner or is the way it is now correct?

Reply:
The second half, the part in the {curly brackets}, is fine. Only the top part needs changing.

Reply:
Welcome! 

The top part is what "users" will use, so that's what needs to reformatted. 

The bottom half is what the keyboard uses, and shouldn't be reformatted.